docs/platform_ccl.rst

```diff
@@ -0,0 +1,297 @@
+CC-Light Plaform Configuration
+------------------------------
+
+`hardware_configuration_cc_light.json <https://github.com/QE-Lab/OpenQL/blob/develop/tests/hardware_config_cc_light.json>`_ available inside the ``tests`` direcotry is an example configuration file for CC-Light platform. 
+
+``cc_light_compiler`` is the backend compiler used for CC-Light platform as specified by ``eqasm_compiler`` field as:
+
+.. code::
+
+    "eqasm_compiler" : "cc_light_compiler",
+
+Next section is ``hardware_settings`` which is used to configure various hardware settings of the platform as shown below:
```

Comment:
Worth specifying what this information will constraint. Will it affect the mapping, the scheduling, etc?

Reply:
updated